blog/journey-to-d14n.md

```diff
@@ -0,0 +1,111 @@
+---
+slug: journey-to-decentralization
+hide_table_of_contents: false
+title: "XMTP: The journey to decentralization"
```

Comment:
I changed this from "Our journey to decentralization" - just to mention and put the focus on XMTP and remove and take the focus off of the word "our"